geonode/resource/models.py

```diff
@@ -37,7 +37,7 @@ class ExecutionRequest(models.Model):
         (STATUS_FINISHED, _("finished")),
     ]
 
-    ACTION_CHOICES = [(v.value, v.value) for v in ExecutionRequestAction]
+    ACTION_CHOICES = [(str(v.value), str(v.value)) for v in ExecutionRequestAction]
```

Comment:
Hi @marthamareal , a question.
How does this constant used inside the ExecutonRequest mode in the action choices break the swagger UI? I'm just curious because I don't understand the reason